e2e/cypress/integration/ui/compliance/01_ssh_scan_job.spec.ts

```diff
@@ -18,100 +18,81 @@ describe('create a manual node ssh scan job and cleanup after', () => {
   const decoded = Base64.decode(Cypress.env('AUTOMATE_ACCEPTANCE_TARGET_KEY'));
 
   it('can create a credential with a username and key', () => {
-    // navigate to credentials create page:
-    // click on settings
-    cy.get('.nav-link').contains('Settings').click();
-
     // we save the route that will be called when we navigate to the page
     // in order to be able to wait for it later
     cy.route('POST', '/api/v0/secrets/search').as('getSecrets');
 
-    // click on node credentials in sidebar
-    cy.get('chef-sidebar-entry').contains('Node Credentials').click();
-    cy.url().should('include', '/settings/node-credentials');
-
-    // wait for data to return
-    cy.wait('@getSecrets');
-
     // click on add credential button to open create form
-    cy.contains('Add Credential').click().then(() => {
-      cy.url().should('include', '/settings/node-credentials/add');
+    cy.contains('Add Credential').click();
+    cy.url().should('include', '/settings/node-credentials/add');
 
-      // fill in name for credential, username, and key
-      cy.get('form input[formcontrolname="name"]').first().type(credName);
-      cy.get('form input[formcontrolname="username"]').first()
-        .type(Cypress.env('AUTOMATE_ACCEPTANCE_TARGET_USER'));
-      cy.get('form textarea[formcontrolname="key"]')
-        .type(decoded);
+    // fill in name for credential, username, and key
+    cy.get('form input[formcontrolname="name"]').first().type(credName);
+    cy.get('form input[formcontrolname="username"]').first()
+      .type(Cypress.env('AUTOMATE_ACCEPTANCE_TARGET_USER'));
 
-      // we save the route that will be called when we navigate to the page
-      // in order to be able to wait for it later
-      cy.route('POST', '/api/v0/secrets').as('createSecret');
+    // we save some time by setting this long string as the value rather than typing it out
+    cy.get('form textarea[formcontrolname="key"]').clear().invoke('val', decoded).trigger('input');
```

Comment:
this cert takes a while to type, and most people would just copy-paste it anyway, so I decided to programmatically set the value instead of using `type` (see [this issue](https://github.com/cypress-io/cypress/issues/1123#issuecomment-485329565))